src/v/debug_bundle/types.h

```diff
@@ -127,6 +127,7 @@ struct debug_bundle_status_data {
     debug_bundle_status status;
     clock::time_point created_timestamp;
     ss::sstring file_name;
+    std::optional<size_t> file_size;
```

Comment:
why is it optional when other fields aren't?

Reply:
`file_size` not provided until after a successful run

Reply:
ah makes sense